.github/sync-repo-settings.yaml

```diff
@@ -3,7 +3,7 @@
 branchProtectionRules:
 # Identifies the protection rule pattern. Name of the branch to be protected.
 # Defaults to `master`
-- pattern: master
+- pattern: '{master,v3.x.x}'
```

Comment:
Hmm, I'm not sure of the syntax -- are you planning that the actual branch name will be `v3.x.x`?  ISTM that `v3` would be clearer:  it is the branch from which `3.x.x` releases will be made.

Reply:
Yes, I was planning `v3.x.x` to be the actual name. `v3` SGTM, though.